xwiki-platform-core/xwiki-platform-oldcore/src/main/java/com/xpn/xwiki/web/ExportAction.java

```diff
@@ -70,6 +75,117 @@
  */
 public class ExportAction extends XWikiAction
 {
+    /**
+     * Used to separate page arguments in excludes parameter
+     */
+    private static String PAGE_SEPARATOR = "&";
+
+    /**
+     * Manage the arguments of the export and provides them in an object to
+     * simplify their usage in the different methods.
+     *
+     * @since 10.9RC1
+     */
+    private class ExportArguments {
+        /**
+         * All patterns that have to be included in the request
+         *
+         * Warning: order of the arguments matters, it's linked with excludedPages
```

Comment:
In that case you can maybe use [`org.apache.commons.lang3.tuple.Pair`](https://commons.apache.org/proper/commons-lang/apidocs/org/apache/commons/lang3/tuple/Pair.html) or a HashMap?